packages/snap-platforms/bigcommerce/src/plugins/pluginAddToCart.ts

```diff
@@ -0,0 +1,26 @@
+import type { Product } from '@searchspring/snap-store-mobx';
+import type { AbstractController } from '@searchspring/snap-controller';
+import type { Next } from '@searchspring/snap-event-manager';
+import { BigCommerceAddToCartConfig } from '../addToCart';
+import { addToCart as addToCartFunction } from '../addToCart';
+import type { AbstractPluginConfig } from '../../../common/src/types';
+
+export type BigCommerceAddToCartPluginConfig = BigCommerceAddToCartConfig &
+	AbstractPluginConfig & {
+		functionOverride?: (products: Product[]) => void;
```

Comment:
I'm not sure if having the functionOverride is necessary - if they don't want to use the default behavior, wouldn't disabling this and using the common plugin function be the same?